### ISSUE TITLE
langchain cannot be installed in a new project

### ISSUE THREAD

**Issue by nipunn1313** [langchain-ai/langchain#10274]
### System Info

npm --version
8.19.4

The langchain it's trying to install is 0.0.144

I think it's because chromadb recently released 1.5.7 and 1.5.8 which added a (conflicting) dep on cohere-ai. Pinning chromadb 1.5.6 works, but that's hardly satisfying.

### Information

- [ ] The official example notebooks/scripts
- [ ] My own modified scripts

### Related Components

- [ ] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

mkdir testproject && cd testproject && npm init -y && npm add langchain

### Expected behavior

Expected to install properly. Instead get
```
npm add langchain
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: testproject@1.0.0
npm ERR! Found: cohere-ai@6.2.2
npm ERR! node_modules/cohere-ai
npm ERR!   peerOptional cohere-ai@"^6.0.0" from chromadb@1.5.8
npm ERR!   node_modules/chromadb
npm ERR!     chromadb@"^1.5.6" from the root project
npm ERR!     peerOptional chromadb@"^1.5.3" from langchain@0.0.144
npm ERR!     node_modules/langchain
npm ERR!       langchain@"*" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional cohere-ai@"^5.0.2" from langchain@0.0.144
npm ERR! node_modules/langchain
npm ERR!   langchain@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by purple-banana** [comment]
This solves it for me:

```
npm i langchain@latest
```

**Comment by jsandlerus** [comment]
Solved in version 0.0.145
